core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3298,6 +3298,9 @@ public FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFile
         mTtlBuckets.insert(inode);
         inode.setLastModificationTimeMs(opTimeMs);
         inode.setTtlAction(options.getTtlAction());
+      } else if (inode.getTtlAction() != options.getTtlAction()) {
+        inode.setLastModificationTimeMs(opTimeMs);
+        inode.setTtlAction(options.getTtlAction());
```

Comment:
Could the logic be consolidated, the last two lines of both if statements are the same?

Reply:
done